draft-ietf-ppm-dap.md

```diff
@@ -829,35 +810,6 @@ coordinated. The difference between the minimum batch size and maximum batch
 size is in part intended to allow room for error, and allow a range of target
 batch sizes.
 
-### Batch Size Considerations {#batch-size}
```

Comment:
I removed this section because:
 * The second & third paragraphs only apply to multiply-collected VDAFs.
 * The first paragraph boils down to "you should probably set `min_batch_size` and `max_batch_size` to the same value, and if you don't the Leader can set batches freely to any size between these two values."

Should we remove the concept of `max_batch_size`?

Reply:
Let's consider removing `max_batch_size` in a separate issue.